presto-parser/src/main/antlr4/io/prestosql/sql/parser/SqlBase.g4

```diff
@@ -308,6 +308,7 @@ primaryExpression
     | identifier '->' expression                                                          #lambda
     | '(' (identifier (',' identifier)*)? ')' '->' expression                             #lambda
     | '(' query ')'                                                                       #subqueryExpression
+    | ARRAY '(' query ')'                                                                 #arrayByTableSubquery
```

Comment:
Is it equivalent to `SELECT array_agg(x) FROM query`?

Reply:
Yes, as a scalar subquery.